reflex-dom-core/src/Reflex/Dom/Builder/Immediate.hs

```diff
@@ -56,7 +57,6 @@ module Reflex.Dom.Builder.Immediate
 #ifndef USE_TEMPLATE_HASKELL
        , phantom2
 #endif
-       , drawChildUpdate
```

Comment:
This is intentional, because drawChildUpdate ought to actually be internal, right?